### PR TITLE
createStatus docs: Include possible "state" fields

### PR DESCRIPTION
**Pull request by brianamarie**
### Why?

With the `createStatus` action, `pending`, `failure`, and `success` can be their own field as well as the options for the `state` field. The current documentation is clear about those as their own fields, but unclear about what the possible options are for the `state` field. 

### What is being changed?

Documentation

---

If adding or updating an action, please verify that you have:
- ~~Added or updated tests, and verified they pass by executing `npm test`~~
- ~~Added or updated code comments, if applicable~~
- ~~Generated up-to-date documentation by executing `npm run doc`, if a schema was added or updated~~

### CODE REVIEW

**Comment by JasonEtco** [comment]
Thanks for the addition @brianamarie! These descriptions are stored in the respective action's schema file:

https://github.com/github/learning-lab-components/blob/1b0453f9f0a2b2275fd39a8996594e798518d922/actions/createStatus/schema.js#L19

Once changed there, we would generate the docs with `npm run doc`. Would you mind making the same adjustment there?

**Comment by brianamarie** [comment]
> Once changed there, we would generate the docs with `npm run doc`. Would you mind making the same adjustment there?

Gladly @JasonEtco! I've pushed up the changes so they're ready for another review once you have a minute. 👀

**Comment by brianamarie** [comment]
There were some other things that were not working as I expected them to with this action, so I'm going to wait to merge this until I figure out more. 

TL;DR - 

- In the before step, I expected this to set a failing status to a specific commit (and PR). Instead, it set an eternally yellow status on that commit.
```yml
- type: createStatus
  state: failure
  sha: 8a649c52d198e6ab0e80692bb660d08b299f479f
```
- I also expected the following to set a passing status on the commit which triggered this action. It did nothing. 
```yml
- type: createStatus      
  state: success
```
- When I added a SHA field, it pointed to the correct commit, but it was still a yellow status.
```yml
- type: createStatus      
  state: success
  sha: '%payload.head_commit.id%'
```

If the action is working as its meant to, I'd like to first understand what I need to do to get the behavior I expect, and then update the documentation to reflect the syntax in this use case.

**Comment by brianamarie** [comment]
Updates - this is causing me some confusion while I'm developing this course, so I'm not sure how to fix the docs. 

Then I try to set `status: failure`, it doesn't work - it goes straight to pending.  When I try to set the status with some logic, it will pass, even if the logic is `1 === 0`. @hectorsector found the following:

and I ran it locally, and it looks like it comes back with a “pending” right away
```
15:13:00.054Z DEBUG github: GitHub request: POST /repos/:owner/:repo/statuses/:sha - 201 Created (installation=805873)
  params: {
    "owner": "hectorsector",
    "repo": "my-amazing-coursessss",
    "context": "before",
    "state": "pending",
    "sha": "73004fc202f9eae3092eab53ac21c76e5b9edf34",
    "baseUrl": "https://api.github.com",
    "request": {
      "timeout": 0
    }
  }
```

I want to figure this out on Friday, but any help @JasonEtco @github/learning-engineering before then would be very appreciated. 🙇

**Comment by brianamarie** [comment]
Thank you! @github/learning-engineering It looks like this has been open a long time, and I don't have permission to merge. (Guessing there's some deploy involved. 😊) Could someone please do the honors and deploy/merge once the tests pass?

**Comment by JasonEtco** [comment]
Thanks for the ping @brianamarie - I thought you were able to merge it 🙈 I'll merge it now, we'll need to pull in the changes to Learning Lab proper which won't happen right away unfortunately. In any case, thank you for the improvement!
